packages/office-ui-fabric-react/src/components/Slider/Slider.scss

```diff
@@ -21,6 +22,49 @@
   box-sizing: border-box;
 }
 
+
```

Comment:
No, these styles also for row slider. The difference is height of the line span.